llama-index-integrations/llms/llama-index-llms-mistral-rs/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.mistral_rs"
+
+[tool.llamahub.class_authors]
+CLASS = "EricLBuehler"
```

Comment:
This should be `MistralRS` instead of `CLASS` :)

Reply:
Done!